Code/Editor/EditorModularViewportCameraComposer.cpp

```diff
@@ -311,4 +345,62 @@ namespace SandboxEditor
                 m_viewportId, &AtomToolsFramework::ModularViewportCameraControllerRequestBus::Events::ClearReferenceFrame);
         }
     }
+
+    void EditorModularViewportCameraComposer::OnTick(const float deltaTime, [[maybe_unused]] AZ::ScriptTimePoint time)
+    {
+        const float delta = deltaTime / ed_cameraDefaultOrbitFadeDuration;
```

Comment:
Since `ed_cameraDefaultOrbitFadeDuration` is a CVAR this code maybe should protect against users entering value 0, to avoid a division by 0.

Reply:
👍